### PR TITLE
Add more details about the HighAvailability param and SpotInstances

### PR DESCRIPTION
**Pull request by Twsouza**
Add more details about the HighAvailability param and SpotInstances. Also reordering alphabetically.

### CODE REVIEW

File: docs/reference/rack-parameters.md [hunk 1]
```diff
@@ -124,6 +124,20 @@ Encrypt secrets with KMS.
...
 Existing VPC ID (if blank, a VPC will be created).
 
+
+### HighAvailability
+
+<div class="alert alert-warning">
+This parameter cannot be changed after the rack is created.
+</div>
+
+Whether to enable High Availability mode. This ensure proper resources redundancy to mitigate system failures.
```
Comment (ddoc) [inline docs/reference/rack-parameters.md]: Whether or not to enable..... Choose between failure resiliency and cost efficiency.

File: docs/reference/rack-parameters.md [hunk 2]
```diff
@@ -179,7 +193,7 @@ Updating parameters... OK
 
 ### InstanceCount
 
-The number of EC2 instances in your Rack cluster.
+The number of EC2 instances in your Rack cluster. It's only used for high available clusters.
```
Comment (ddoc) [inline docs/reference/rack-parameters.md]: This parameter is only used for clusters with HighAvailablity = true

File: docs/reference/rack-parameters.md [hunk 3]
```diff
@@ -124,6 +124,20 @@ Encrypt secrets with KMS.
...
+
+### HighAvailability
+
+<div class="alert alert-warning">
+This parameter cannot be changed after the rack is created.
+</div>
+
+Whether to enable High Availability mode. This ensure proper resources redundancy to mitigate system failures.
+
+If High Availability is true, the [InstanceCount](#instancecount) is used as initial cluster size. If false, the `NoHaInstanceCount` is used as initial cluster size. Both can be scaled to 1000 instances.
```
Comment (ddoc) [inline docs/reference/rack-parameters.md]: If HighAvailability is set to true

**Review by ddoc — CHANGES_REQUESTED** [review]

File: docs/reference/rack-parameters.md [hunk 4]
```diff
@@ -179,7 +193,7 @@ Updating parameters... OK
 
 ### InstanceCount
 
-The number of EC2 instances in your Rack cluster.
+The number of EC2 instances in your Rack cluster. This parameter is only used for clusters with HighAvailablity as true.
```
Comment (heronrs) [inline docs/reference/rack-parameters.md]: `HighAvailablity=true` i think you missed this one.

File: docs/reference/rack-parameters.md [hunk 5]
```diff
@@ -124,6 +124,20 @@ Encrypt secrets with KMS.
...
 Existing VPC ID (if blank, a VPC will be created).
 
+
+### HighAvailability
+
+<div class="alert alert-warning">
+This parameter cannot be changed after the rack is created.
+</div>
+
+Whether to enable High Availability mode, choose between failure resiliency and cost efficiency. This ensure proper resources redundancy to mitigate system failures.
```
Comment (heronrs) [inline docs/reference/rack-parameters.md]: `Whether or not`

File: docs/reference/rack-parameters.md (hunk 4 above)
Comment (Twsouza) [reply docs/reference/rack-parameters.md]: Yeah, I thought that this way would be more clear, I changed it in the last commit.

**Pull request merged**